components/p3a/brave_p3a_service.cc

```diff
@@ -48,13 +52,20 @@ namespace {
 constexpr int32_t kSuspendedMetricValue = INT_MAX - 1;
 constexpr uint64_t kSuspendedMetricBucket = INT_MAX - 1;
 
-constexpr char kLastRotationTimeStampPref[] = "p3a.last_rotation_timestamp";
+constexpr char kLastTypicalRotationTimeStampPref[] =
+    "p3a.last_rotation_timestamp";
+constexpr char kLastExpressRotationTimeStampPref[] =
+    "p3a.last_express_rotation_timestamp";
+constexpr char kDynamicMetricsDictPref[] = "p3a.dynamic_metrics";
 
 constexpr char kP3AServerUrl[] = "https://p3a-json.brave.com/";
+constexpr char kP3ACreativeServerUrl[] = "https://p3a-creative.brave.com/";
```

Comment:
Please add an entry for this hostname in https://github.com/brave/brave-core/blob/932ae258134827d78d125a4c3676fd7dca5f2db6/chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc#L425

Reply:
didn't know about this, thanks